app/app/settings.py

```diff
@@ -154,3 +154,52 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
+if DEBUG:
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "handlers": {
+            "console": {
+                "class": "logging.StreamHandler",
+            },
+        },
+        "root": {
+            "handlers": ["console"],
+            "level": "DEBUG",
+        },
+    }
+else:
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "handlers": {
+            "console": {
+                "class": "logging.StreamHandler",
+            },
+            "file": {
+                "level": os.environ.get("LOGGING_HANDLERS_LEVEL", "WARNING"),
+                "class": "logging.FileHandler",
+                "filename": os.environ.get("LOGGING_FILE", "logging/debug.log"),
```

Comment:
Imagine `LOGGING_FILE=general/debug.log`. Now you can't as easily mount a volume for the logging, since it will interfere with the application directories, unless someone knows to configure a file volume.

If someone misconfigures this by providing a directory name, they might overwrite project files. It must go into a directory that is documented in the list of volumes you added in the README so that there is no risk of overwriting something else, or complicating deployment unnecessarily. In other words, I propose something along the lines of:
```python
                "filename":  BASE_DIR / "logging" / os.environ.get("LOGGING_FILE", "debug.log"),
```
I guess someone could still pass `../general/debug.log`, but then they're actively trying to subvert their own system :-)

Reply:
changed default to logging/debug.log